front/src/routes/scene/edit-scene/triggers/MessageQueueReceivedTrigger.jsx

```diff
@@ -0,0 +1,32 @@
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+import { Text, Localizer } from 'preact-i18n';
+
+class MessageQueueReceived extends Component {
+  updateTopicName = e => {
+    this.props.updateTriggerProperty(this.props.index, 'topic', e.target.value);
+  };
+
+  render({}, {}) {
+    return (
+      <div>
+        <div class="form-group">
+          <label className="form-label">
+            <Text id="editScene.triggersCard.messageQueueReceived.topicLabel" />
+          </label>
+          <Localizer>
+            <input
+              type="text"
+              value={this.props.trigger.topic}
+              onInput={this.updateTopicName}
+              className="form-control"
+              placeholder={<Text id="editScene.triggersCard.messageQueueReceived.topicPlaceholder" />}
+            />
+          </Localizer>
```

Comment:
User should be able to check for the content of the message as well, because in the use case specified in the feature request, the goal was to be able to use Frigate with Gladys ( https://community.gladysassistant.com/t/nouveau-declencheur-arrivee-d-un-message-mqtt/8127 ) 

In Frigate, the message looks like this:

![Screenshot 2023-10-26 at 10 28 18](https://github.com/GladysAssistant/Gladys/assets/7365207/285e95dc-bb85-4d60-9fd2-d66bdbecadf6)

(Doc: https://docs.frigate.video/integrations/mqtt/#frigatecamera_namemotion ) 

So being able to check for a specific text in the message is important :) 

(It should not be mandatory, if empty, should not be verified)

Reply:
I have added this feature now :)